cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/repositories.go

```diff
@@ -72,6 +72,12 @@ func (s *Server) newRepo(ctx context.Context, repo *HelmRepository) (*corev1.Pac
 			return nil, err
 		}
 	}
+	// Copy secret to global namespace if needed
```

Comment:
I would add here a reference to the issue, maybe #5129 or the previous one, in order to have the full context once we get back to this line in the future.